crates/api_models/src/disputes.rs

```diff
@@ -49,6 +49,11 @@ pub struct DisputeResponse {
     pub merchant_connector_id: Option<String>,
 }
 
+pub struct FileInfo {
```

Comment:
Why is this struct defined in api_models? We can consider this moving into hyperswitch domain models